fluss-lake/fluss-lake-lance/src/main/java/com/alibaba/fluss/lake/lance/LanceConfig.java

```diff
@@ -130,24 +130,8 @@ public static WriteParams genWriteParamsFromConfig(LanceConfig config) {
         return builder.build();
     }
 
-    private static Map<String, String> genStorageOptions(LanceConfig config) {
```

Comment:
looks like this is copied from the recent changes in the Spark connector. I am still refactoring the Lance config, currently I just pass the whole thing as storage options because there are many possible config keys and it is hard to enumerate all of them. But there should probably be a better way to do it so it does not mix the write params and storage options. I can put up a separated PR here after I figure out how to do that in Spark.